openmdao/docs/feature_reference/building_blocks/solvers/backtracking/bounds_enforce.rst

```diff
@@ -3,12 +3,73 @@
 Bounds Enforce
 **************
 
-Description of BoundsEnforce.
+The BoundsEnforce linesearch only backtracks until variables that violate their upper and lower bounds.
```

Comment:
You've left of the `LS` here, but not for Armijo... should be consistent?  Also.. I think "line search" should be two words, not hyphenated, per https://en.wikipedia.org/wiki/Backtracking_line_search (there are multiple variations on this through these docs)

Reply:
Fixed, we must have changed the name and not updated the docs.